packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -608,3 +608,16 @@ exports.createNodesFromEntities = ({ entities, createNode }) => {
     })
   })
 }
+
+exports.createUrlPathsFromLinks = entities =>
+  entities.map(e => {
+    if (e.__type === `wordpress__PAGE` || e.__type === `wordpress__POST`) {
```

Comment:
This won't handle custom post types. I wonder if we instead could check:
```js
if (e.link && !e.path) 
```
to check if link is on entity and just sanity check to not override path field if it already exists (someone or some plugin enriching REST endpoint on wordpress side)

Reply:
Thanks for your feedback. Also the sanity check makes totally sense.

I was conservative regarding the types, as the current Wordpress project I use for testing only has normal pages and posts. 
But I should have a deeper look into the example data stemming from the official Gatsby demo wordpress site. 

I’ll try it tomorrow :)

Reply:
@pieh, I added your suggestion.